essentials/strings/escaping-special-characters/quiz/index.md

```diff
@@ -4,7 +4,7 @@ title: 'Quiz: A backslash in the output'
 
 {% include menu.html %}
 
-From the below strings, check those that will have precisely one slash in the output if you print them.
+From the below strings, check those that will have precisely one backslash `\` in the output if you print them.
```

Comment:
Clarify because there are many forward slashes in the output. The quiz title is correct